ReactAndroid/src/main/java/com/facebook/react/modules/core/DeviceEventManagerModule.java

```diff
@@ -53,6 +56,15 @@ public void emitHardwareBackPressed() {
   }
 
   /**
+   * Sends an event to the JS instance that a new intent was received.
+   */
+  public void emitNewIntent(Uri uri) {
+    getReactApplicationContext()
+        .getJSModule(RCTDeviceEventEmitter.class)
+        .emit("openURL", uri.toString());
```

Comment:
@mkonicek The event is called `openURL` on iOS, so keeping same name means no extra handling for android is needed on JS side.